src/styles/_catalog.scss

```diff
@@ -14,9 +14,17 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+    transition: 300ms;
```

Comment:
Add properties to which you apply transition

Reply:
I can't do it here. I have more than one property for this transition time and it doesn't work otherwise.

Reply:
Try something like this: transition: transform 0.3s, color 0.3s;

Reply:
Already figured. Just had to use coma